content/en-us/education/battle-royale-series/coding-the-game-loop.md

```diff
@@ -165,13 +165,13 @@ With a **repeat loop**, the code in the loop will run at least once. Unlike a wh
    end
    ```
 
-3. In the repeat loop, add a print statement saying the intermission is starting. Use `wait()` to pause for the intermission using `intermissionDuration` from GameSettings.
+3. In the repeat loop, add a print statement saying the intermission is starting. Use `task.wait()` to pause for the intermission using `intermissionDuration` from GameSettings.
```

Comment:
Adding `Libary.` makes the code automatically link to the [reference page](https://create.roblox.com/docs/reference/engine/libraries/task#wait) (see [STYLE.md](https://github.com/Roblox/creator-docs/blob/main/STYLE.md#links)):

```suggestion
3. In the repeat loop, add a print statement saying the intermission is starting. Use `Library.task.wait()` to pause for the intermission using `intermissionDuration` from GameSettings.
```